themes/classic/templates/catalog/listing/product-list.tpl

```diff
@@ -59,9 +59,13 @@
         </div>
 
       {else}
+        <div id="js-product-list-top"></div>
```

Comment:
These new `<div>` tags need to be ported on the StarterTheme, don't they?

Reply:
StarterTheme is abandonned.

Reply:
Well, the repository needs to be updated accordingly.